activating the news page (#7)
djangocon/2025.djangocon.us: src/_content/posts/call-for-proposals.md

---

> author: communications Team

Reviewer: ```suggestion
author: Communications Team
```

Reviewer: ✏️ Fix case

category: General
published_datetime: 2025-03-04 06:00:00
cover:
    url: /assets/img/2025/54089577525_de6f9bcb72_w.jpg

>     alt: "Sheena O'Connel speaking at DjangoCon US 2024"

Reviewer: ```suggestion
    alt: "Sheena O'Connell speaking at DjangoCon US 2024"
```

Reviewer: ✏️ fix typo

title: "The Call for Proposals for DjangoCon US 2025 Is Now Open!"
---

> We are excited that DjangoCon US 2025 is now open for talk submissions: [call for proposals](https://pretalx.com/djangocon-us-2025/cfp)! The deadline for submissions is April 27th, 2025 at [Noon CST](https://time.is/1200PM_27_April_2025_in_Chicago?DjangoCon_US_CFP_Closes).

Reviewer: ```suggestion
We are excited that DjangoCon US 2025 is now open for talk submissions: [call for proposals](https://pretalx.com/djangocon-us-2025/cfp)! The deadline for submissions is April 27th, 2025 at [Noon CDT](https://time.is/1200PM_27_April_2025_in_Chicago?DjangoCon_US_CFP_Closes).
```

Time zone fix

<!-- excerpt -->

We invite you to submit your proposal no matter your background or experience level with Django. Proposals can be from a wide range of topics; non-Django and community topics are welcome. You can look at our [talk lineup](https://2024.djangocon.us/news/announcing-lineup/) from last year for reference.

> We fancy first-timers! If you haven’t spoken at a conference or given a tutorial before, this is your invitation to do so. Don’t let the idea that you’re not famous or an expert stop you from submitting. It certainly won’t stop us from selecting your talk or tutorial and it won’t stop the audience from enjoying it! If you still need help decding on a topic take a look at our own Drew Wentzel's [blog post](https://winstel.dev/2025/02/12/talk-ideas-for-dcus-25/) for some ideas.

Reviewer: ```suggestion
We fancy first-timers! If you haven’t spoken at a conference or given a tutorial before, this is your invitation to do so. Don’t let the idea that you’re not famous or an expert stop you from submitting. It certainly won’t stop us from selecting your talk or tutorial and it won’t stop the audience from enjoying it! If you still need help decding on a topic take a look at our own Drew Winstel's [blog post](https://winstel.dev/2025/02/12/talk-ideas-for-dcus-25/) for some ideas.
```

That's a new way to misspell my name (and I've seen a lot)! 😆

Author: How in the world did I do that!? LOL.

> Plus there are perks! Presenters get free admission to DjangoCon US! Grants to assist with your travel and lodging expenses are available as well. Fill out the [Opportunity Grants](https://forms.gle/Pi12J6vFQHq2CSAy5) form by April 24th, 2024. Decision notifications will be sent by June 14, 2024.

Reviewer: ```suggestion
Plus there are perks! Presenters get free admission to DjangoCon US! Grants to assist with your travel and lodging expenses are available as well. The form for Opportunity Grants will be released in the very near future.
```

For more information on talk and tutorial formats, please check out our [speaker information page](https://2025.djangocon.us/speaking/).

We want everyone attending DjangoCon US to feel safe, welcome, and included. To that end, we have a [Code of Conduct](https://2025.djangocon.us/conduct/) for all speakers and attendees.

> If you have questions feel free [to contact us](mailto:hello@djangocon.us).

Reviewer: ```suggestion
If you have questions, feel free to [contact us](mailto:hello@djangocon.us).
```
